Watchman.Discord/Areas/Help/Factories/ArgumentInfoFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Models;
+using Watchman.DomainModel.Help;
+
+namespace Watchman.Discord.Areas.Help.Factories
+{
```

Comment:
Nie wiem jak w tej sytuacji
Ale pamiętaj że takie rzeczy możesz też robić przez konstruktor

Reply:
Możliwe że tutaj jest jakiś powód dlaczego jest fabryka

Reply:
Ok, w tym przypadku jest dobrze

Reply:
Zamiast Factory myśle że lepiej byłoby to nazwać Mapper
Ale jest ok, najwyżej później sie zmieni